client/src/components/library/admin/BookForm.js

```diff
@@ -2,7 +2,7 @@ import React, { Component } from 'react';
 import classnames from 'classnames';
 import PropTypes from 'prop-types';
 import { connect } from 'react-redux';
-import { saveBook } from './../../../actions/createActions';
+import { saveBook } from './../../../actions/bookActions';
```

Comment:
Move `actions` to component directory.

`import {saveBook} from `/app/components/Book/actions``

Reply:
ok. so actions folder should be a subfolder in components directory right?

Reply:
That's right.